### PR TITLE
MINOR: Added scripts to automate Vagrant setup for system tests

### PR DESCRIPTION
**Pull request by granders**
Updated testing README accordingly.

### CODE REVIEW

**Comment by granders** [comment]
@ewencp @gwenshap 
Hopefully this starts to make system test setup a little more out-of-the-box

**Comment by gwenshap** [comment]
thats awesome! I'm still testing it, but looks awesome :)

File: vagrant/package-base-box.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+local_vagrantfile=Vagrantfile.local
+if [ -e $local_vagrantfile ]; then
+    mv $local_vagrantfile $backup_vagrantfile
+fi
+echo "num_workers = 1" > $local_vagrantfile
+echo "num_brokers = 0" >> $local_vagrantfile
+echo "num_zookeepers = 0" >> $local_vagrantfile
+vagrant up
+up_success=$?
+if [ $up_sucess != 0 ]; then
```
Comment (ewencp) [inline vagrant/package-base-box.sh]: Got this output when I ran this:

```
vagrant/package-base-box.sh: line 32: [: !=: unary operator expected
Packaging worker1...
```

I think you just have a typo: `up_sucess` -> `up_success`

File: vagrant/package-base-box.sh [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    echo "Failed to bring up a template vm, please try running again."
+    # restore original Vagrantfile.local
+    rm -f $local_vagrantfile
```
Comment (ewencp) [inline vagrant/package-base-box.sh]: This is repeated below. You can create a function to do this cleanup.

File: tests/README.md [hunk 3]
```diff
@@ -14,51 +14,30 @@ https://cwiki.apache.org/confluence/display/KAFKA/tutorial+-+set+up+and+run+Kafk
...
 * Install Virtual Box from [https://www.virtualbox.org/](https://www.virtualbox.org/) (run `$ vboxmanage --version` to check if it's installed).
 * Install Vagrant >= 1.6.4 from [http://www.vagrantup.com/](http://www.vagrantup.com/) (run `vagrant --version` to check if it's installed).
-* Install Vagrant Plugins:
+* Install ducktape (system testing command-line tool and library)
 
-        # Required
-        # Note that vagrant-hostmanager v1.6.0 and up breaks our Vagrant scripts
-        $ vagrant plugin install vagrant-hostmanager --plugin-version 1.5.0
-        $ vagrant plugin install vagrant-cachier
+        $ pip install ducktape
```
Comment (ewencp) [inline tests/README.md]: Does this need to be a specific version? Should just get the dependencies via `setup.py`?

**Comment by ewencp** [comment]
@granders Found a couple of minor issues, but otherwise worked fine for me. That said, I had a bunch of stuff installed already so I'm not sure about the vagrant and plugin install parts...

**Comment by granders** [comment]
@gwenshap @ewencp 
Added a few small modifications per feedback.

**Comment by ewencp** [comment]
@gwenshap @granders Looks good to me now.

**Pull request closed without merging**